src/System.Numerics.Experimental/Half.v1.cs

```diff
@@ -241,39 +241,54 @@ public static bool IsSubnormal(Half value)
                 && ((absValue & ExponentMask) == 0);    // is subnormal (has a zero exponent)
         }
 
-        public static Half Parse(string s, NumberStyles style = DefaultParseStyle, IFormatProvider formatProvider = null)
+        public static Half Parse(string s)
+        {
+            return (Half)float.Parse(s, style: DefaultParseStyle, provider: null);
```

Comment:
This isn't quite right and may result in some differences (iirc). Fixing the actual parsing code should be simple though as it "should" just be updating a few numbers representing the buffer size and max digits to parse.